frontend/src/concepts/topology/PipelineTaskEdge.tsx

```diff
@@ -13,6 +13,7 @@ interface PipelineTaskEdgeProps {
 }
 
 const PipelineTaskEdge: React.FC<PipelineTaskEdgeProps> = ({ element, ...props }) => {
+  // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
   const edge = element as Edge;
```

Comment:
@rsun19 hmmm this may not be fixable in topology. My suggestion is causing other issues.

Instead we can use `isEdge` from the topology package to first ensure it is the correct type

```
if (!isEdge(element)) {
  throw new Error(...);
}
```